test/image/mocks/colorscale_template.json

```diff
@@ -0,0 +1,157 @@
+{
+  "data": [{
+    "x": [
+      50076,
+      -42372,
+      -19260,
+      3852,
+      26964,
+      -65484,
+      -42372,
+      -19260,
+      3852,
+      26964,
+      -88596,
+      -65484,
+      -42372,
+      -19260,
+      3852,
+      26964,
+      50076,
+      73188,
+      -65484,
+      -42372,
+      -19260,
+      3852,
+      26964,
+      50076,
+      -42372,
+      -19260,
+      3852,
+      26964,
+      -88596,
+      -65484,
+      -42372,
+      -19260,
+      3852,
+      26964,
+      50076,
+      73188,
+      -88596,
+      -65484,
+      -42372,
+      -19260,
+      3852,
+      26964,
+      50076,
+      73188
+    ],
+    "y": [
+      51851.8,
+      77841.4,
+      77841.4,
+      77841.4,
+      77841.4,
+      51851.8,
+      51851.8,
+      51851.8,
+      51851.8,
+      51851.8,
+      -26117,
+      -26117,
+      -26117,
+      -26117,
+      -26117,
+      -26117,
+      -26117,
+      -26117,
+      -52106.6,
+      -52106.6,
+      -52106.6,
+      -52106.6,
+      -52106.6,
+      -52106.6,
+      -78096.2,
+      -78096.2,
+      -78096.2,
+      -78096.2,
+      -127.4,
+      -127.4,
+      -127.4,
+      -127.4,
+      -127.4,
+      -127.4,
+      -127.4,
+      -127.4,
+      25862.2,
+      25862.2,
+      25862.2,
+      25862.2,
+      25862.2,
+      25862.2,
+      25862.2,
+      25862.2
+    ],
+    "z": [
+      4.361856,
+      4.234497,
+      4.321701,
+      4.450315,
+      4.416136,
+      4.210373,
+      4.32009,
+      4.246728,
+      4.293992,
+      4.316364,
+      3.908434,
+      4.433257,
+      4.364234,
+      4.308714,
+      4.275516,
+      4.126979,
+      4.296483,
+      4.320471,
+      4.339848,
+      4.39907,
+      4.345006,
+      4.315032,
+      4.295618,
+      4.262052,
+      4.154291,
+      4.404264,
+      4.33847,
+      4.270931,
+      4.032226,
+      4.381492,
+      4.328922,
+      4.24046,
+      4.349151,
+      4.202861,
+      4.256402,
+      4.28972,
+      3.956225,
+      4.337909,
+      4.31226,
+      4.259435,
+      4.146854,
+      4.235799,
+      4.238752,
+      4.299876
+    ],
+    "type": "heatmap",
+    "autocolorscale": true
+  }],
+  "layout": {
+    "template": {
+      "layout": {
+        "title": "heatmap template",
+        "colorscale": {
```

Comment:
What if we put `autocolorscale: true` in the `template` container, does it still work?

Reply:
Good call! I modified the mock in dca4350